index.html

```diff
@@ -4,66 +4,67 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link href="assets/CSS/header-style.css" rel="stylesheet">
-    <link href="assets/CSS/content-style.css" rel="stylesheet">
-    <link href="assets/CSS/footer-style.css" rel="stylesheet">
+    <link href="/assets/CSS/style.css" rel="stylesheet">
     <title>Manar Portofolio</title>
 </head>
 <body>
    <header >
        <div class="flex-container">
-           <section class="logo">
-            <a href="https://www.microverse.com">
-                <img src="assets/images/toolbar-mobile.png" alt="image icon logo"></a>
-           </section>
-            <section class="empty-section"></section>
-            <section class="flex-container-ul">
-                     <ul>
-                     <li><a href="#Portofolio.html">Portofolio</a></li>
-                     <li><a href="#about.html">About</a></li>
-                     <li><a href="#Contact.html">Contact</a></li>
-                     </ul>
-                    </section>
-               </div>
+           
+             <nav>
+                 <div class="logo">
+                   <h1><a href="#">My Logo</a></h1>
+                   <a class="hamburger-menue" href="#">
```

Comment:
- [Optional] Nicely done, kindly add a little `right-margin` to the hamburger menu so it aligns with what you have in the Figma Design